### ISSUE TITLE
PID file can contain invalid content

### ISSUE THREAD

**Issue by max0d41** [circus-tent/circus#866]
It seems that it is possible that the PID file contains more than one PID and/or some other trash data. When this happens the PID file needs to be manually deleted/cleaned up. This error prevents circusd from starting.
I got this exception more than once.

```
Traceback (most recent call last):
  File "/usr/local/bin/circusd", line 9, in <module>
    load_entry_point('circus==0.12.0', 'console_scripts', 'circusd')()
  File "/usr/local/lib/python2.7/dist-packages/circus/circusd.py", line 137, in main
    pidfile.create(os.getpid())
  File "/usr/local/lib/python2.7/dist-packages/circus/pidfile.py", line 18, in create
    oldpid = self.validate()
  File "/usr/local/lib/python2.7/dist-packages/circus/pidfile.py", line 66, in validate
    wpid = int(f.read() or 0)
ValueError: invalid literal for int() with base 10: '503\n7\n'
```

**Comment by morganbohn** [comment]
Hi,

I had the same exception : 

Traceback (most recent call last):
  File "/usr/local/bin/circusd", line 9, in <module>
    load_entry_point('circus==0.12.1', 'console_scripts', 'circusd')()
  File "/usr/local/lib/python2.7/dist-packages/circus/circusd.py", line 137, in main
    pidfile.create(os.getpid())
  File "/usr/local/lib/python2.7/dist-packages/circus/pidfile.py", line 18, in create
    oldpid = self.validate()
  File "/usr/local/lib/python2.7/dist-packages/circus/pidfile.py", line 65, in validate
    wpid = int(f.read() or 0)
ValueError: invalid literal for int() with base 10: '452\n4\n'

I've deleted the .circus.pid file and all works fine now. Thanks for the tips.

**Comment by k4nar** [comment]
Thanks for the report, I'll try to look into it. The weird part is that the number after the first `\n` is very unlikely to be a PID (`7` and `4` in your cases).

**Comment by k4nar** [comment]
I have a fix for that waiting in #984, it should be in Circus 0.14.
